isis/src/system/objs/KernelDb/KernelDb.cpp

```diff
@@ -586,10 +586,10 @@ namespace Isis {
     QString instrument = "";
 
     if (grp.hasKeyword("StartOffset")){
-      startOffset = (double) grp["StartOffset"];
+      startOffset = (double) grp["StartOffset"] + 0.001;
```

Comment:
Is increasing start offset necessary?  It seems like truncating at thousandths would still allow the comparison to succeed since 100.123 <= 100.123xxx.

Reply:
It's the opposite. The issue we are running into is 100.123xxx <= 100.123. So adding 0.001 forces it to be 100.122xxx so that the comparison passes.